addons/mail/wizard/mail_compose_message.py

```diff
@@ -161,6 +158,15 @@ def _compute_render_model(self):
         for composer in self:
             composer.render_model = composer.model
 
+    # Onchanges
+
+    @api.onchange('template_id')
+    def onchange_template_id_wrapper(self):
```

Comment:
```suggestion
    def _onchange_template_id_wrapper(self):
```
While you're moving it, apply guidelines fully :D